fuzzers/int_maketodo.py

```diff
@@ -126,22 +127,26 @@ def maketodo(
     '''
 
     todos, tile_type = load_pipfile(pipfile, verbose=verbose)
-    verbose and print('%s: %u entries' % (pipfile, len(todos)))
+    verbose and print(
+        '%s: %u entries' % (pipfile, len(todos)), file=sys.stderr)
     if not todos:
-        verbose and print('%s: %u entries, done!' % (pipfile, len(todos)))
+        verbose and print(
+            '%s: %u entries, done!' % (pipfile, len(todos)), file=sys.stderr)
         return
 
-    verbose and print("pipfile todo sample: %s" % list(todos)[0])
+    verbose and print(
+        "pipfile todo sample: %s" % list(todos)[0], file=sys.stderr)
 
     if 0 and verbose:
```

Comment:
```suggestion
    if verbose:
```

Currently this condition is always false and quite useless, probably it would be better to remove the zero.

Reply:
Thank you for the review. I have pushed the requested changes. Should I squash?